stages/org.osbuild.selinux.meta.json

```diff
@@ -33,9 +33,40 @@
         }
       ],
       "properties": {
+        "target": {
+          "oneOf": [
+            {
+              "type": "string",
+              "description": "Target path, if a mount",
+              "pattern": "^mount://.+"
+            },
+            {
+              "type": "string",
+              "description": "Target path, if a tree",
+              "pattern": "^tree://.+"
+            }
+          ],
+          "default": "tree:///"
+        },
         "file_contexts": {
-          "type": "string",
-          "description": "Path to the active SELinux policy's `file_contexts`"
+          "description": "Path to the active SELinux policy's `file_contexts`",
+          "anyOf": [
+            {
+              "type": "string",
+              "pattern": "^mount://.+"
+            },
+            {
+              "type": "string",
+              "pattern": "^tree:///"
+            },
+            {
+              "type": "string",
+              "pattern": "^input://.+"
+            },
+            {
+              "type": "string"
+            }
+          ]
```

Comment:
This is a bit of a funny situation here.  The patterns are essentially useless, because there's always a plain `string` there with no pattern.  There's no string that would fail this validation.

Perhaps these work for documenting valid or suggested forms of the path, but I think having this in the schema can be misleading.  It might be best to change the description instead:
```
Path to the active SELinux policy's `file_contexts`. Supports `tree://`, `mount://`, and `input://` schemes. Plain paths imply `tree://`.
```

Reply:
Yes, i knew it,  put all here for visibility. Nevermind, changed to plain `string`